### DISCUSSION TITLE
Gotenberg 8.0.3 | Security

### DISCUSSION THREAD

**Discussion by rahulg24** [gotenberg/gotenberg#795] [Q&A]
Version 8.0.3 contains the attached security vulnerabilities. Could someone please provide a more secure version or address these vulnerabilities?
[Gotenberg_Image_Scanning_Report.xlsx](https://github.com/gotenberg/gotenberg/files/14219850/Gotenberg_Image_Scanning_Report.xlsx)

**Comment by gulien** [comment]
Please open a PR and use the dedicated channel as described here https://github.com/gotenberg/gotenberg?tab=security-ov-file.

Also, please chill out with "vulnerabilities".
